src/cpp/client/secure_credentials.cc

```diff
@@ -61,6 +62,7 @@ std::shared_ptr<Credentials> WrapCredentials(grpc_credentials* creds) {
 }  // namespace
 
 std::shared_ptr<Credentials> GoogleDefaultCredentials() {
+  GrpcLibrary init;  // To call grpc_init().
```

Comment:
Shouldn't we do that in `grpc_google_default_credentials_create` instead? What happens if we just use the C API?